app/scripts/lib/metrics.js

```diff
@@ -14,6 +13,7 @@
  * Metrics are automatically sent to the server on window.unload
  * but can also be sent by calling metrics.flush();
  */
+'use strict';
```

Comment:
@vladikoff @shane-tomlinson @zaach What's the consensus of putting `'use strict';` immediately after the copyright headers versus after the comment?